### DISCUSSION TITLE
Type of key and value

### DISCUSSION THREAD

**Discussion by abolfazlrastegar** [filamentphp/filament#11790] [Ideas]
how can require number type for keys or values or both?

**Comment by leandrocfe** [comment]
Should it be a question Q&A?
Could you provide more info please?

**Comment by abolfazlrastegar** [comment]
What should I do so that when using the KeyValue, only the numbers for the input of Key and Valueare entered?

`use Filament\Forms\Components\KeyValue;
 
KeyValue::make('meta')

`
